dart/README.md

```diff
@@ -0,0 +1,13 @@
+# FlatBuffers for Dart
```

Comment:
This is the content that would show up on pub.dartlang.org.  I can link it to the specific Dart documentation, but it shouldn't be the exact content of DartUsage.md because that has some doxygen specific stuff that wouldn't render properly on pub.dartlang.org

Reply:
Ok, that's fine, leave it here then.